sale_procurement_group_by_line/tests/test_sale_procurement_group_by_line.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2013-2014 Camptocamp SA - Guewen Baconnier
+# © 2016-20 ForgeFlow S.L. (https://www.forgeflow.com)
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleProcurementGroupByLine(TransactionCase):
+    def setUp(self):
```

Comment:
Hi, as @BT-anieto  says, can you change it to SetUpClasee?

Reply:
hello @HugoCordobaLeal !! done, i updated the unit tests.
@BT-anieto FYI